JPF/bad_buildings.sql

```diff
@@ -71,7 +105,7 @@ left join
 on
 	c.complaintid = cp.complaintid
 left join 
-	hpd_buildings b
+	hpd_buildings_temp b
 on 
 	c.buildingid = b.buildingid
 where 
```

Comment:
What are these category ids? I don't know if SQL has inline comments, but it'd be nice to include the id name in a comment.